juriscraper/opinions/united_states/state/tenn.py

```diff
@@ -46,7 +46,7 @@ def _process_html(self):
                     "url": url,
                     "name": name,
                     "docket": rows[1],
-                    "judges": rows[2].split(": ")[1],
+                    "judge": rows[2].split(": ")[1],
```

Comment:
the field is judges not judge right.  Judges are the panel of judges of hear the case while author is the judge who wrote the opinion?

Reply:
That difference is useful in Courtlistener. There is no field like OpinionCluster.judge or Opinion.judge in Courtlistener, so I think the difference goes between Opinion.author_str and Opinion.joined_by and OpinionCluster.judges

In Juriscraper, OpinionSiteLinear picks up the "judge" key and transforms it to the "judges" key, which is the only one that it is prepared to used, anyway

https://github.com/freelawproject/juriscraper/blob/e5684219ed08c0ff25d75a90635f4ca87183707f/juriscraper/OpinionSiteLinear.py#L62-L63

This is all kind of confusing, but I am just trying to adhere to the conventions used. OpinionSiteLinear takes a singular noun and transform it to plurals, which what AbstractSite is prepared to return